tests/integration/test_validate_call.py

```diff
@@ -754,9 +754,9 @@ def test_call_by_split_read_shift_overlap(self):
 
         self.assertEqual(101, event.break1.start)
         self.assertEqual(101, event.break1.end)
-        self.assertEqual(503, event.break2.start)
-        self.assertEqual(503, event.break2.end)
-        self.assertEqual('', event.untemplated_seq)
+        self.assertEqual(501, event.break2.start)
+        self.assertEqual(501, event.break2.end)
+        self.assertEqual(None, event.untemplated_seq)
```

Comment:
why is the untemplated seq none? can we no longer resolve it?

Reply:
I'm not exactly sure what happened to this test case...

Reply:
Looks like the reads that were added are no longer sufficient to resolve the untemplated_seq (as they are closer to the 501 breakpoint vs the 503 breakpoint)

Reply:
ok that makes sense. For our reference later this has to do with us adding reads through the add method on the call class instead of adding them directly to the inner dicts

Reply:
however this does mean that this test case no longer covers the case it was intended for, can you add more reads so that it does? iirc this was meant to test when the breakpoints have sequence in common and the caller has to assign it to one of the breakpoints not both. It should assign it to the first breakpoint for consistency. An example would be

break1
```
ATGccg
```
break2
```
attCCATATA
```

The above breakpoints would be shifted to
```
ATGCCg
```

```
attccATATA
```

Reply:
fixed now